builtin/credential/radius/backend_test.go

```diff
@@ -38,6 +38,11 @@ func prepareRadiusTestContainer(t *testing.T) (func(), string, int) {
 		ContainerName: "radiusd",
 		Cmd:           []string{"-f", "-l", "stdout"},
```

Comment:
This will give you debug info from `radiusd`:

```suggestion
		Cmd:           []string{"-f", "-l", "stdout", "-X"},
```

See: https://freeradius.org/radiusd/man/radiusd.html